packages/abi-coder/src/AbiCoder.ts

```diff
@@ -2,11 +2,11 @@ import { ResolvedAbiType } from './ResolvedAbiType';
 import type { DecodedValue, InputValue, Coder } from './encoding/coders/AbstractCoder';
 import { getCoderForEncoding } from './encoding/strategies/getCoderForEncoding';
 import type { EncodingOptions } from './types/EncodingOptions';
-import type { JsonAbi, JsonAbiArgument } from './types/JsonAbi';
+import type { JsonAbiOld, JsonAbiArgument } from './types/JsonAbi';
 
 export abstract class AbiCoder {
   static getCoder(
-    abi: JsonAbi,
+    abi: JsonAbiOld,
```

Comment:
We should stop exporting the `AbiCoder` class.

Reply:
The encode/decode methods should also be dropped, correct?

Reply:
The `AbiCoder` [isn't exported](https://github.com/FuelLabs/fuels-ts/pull/2862/files#diff-ec3377ee7d03f8d048627e26e24db0def496fd4bdc9c737927a40f04b4b39dfdL17) anymore. Now it's an internal detail of the package that works with the old ABI and that we can refactor however we want.

Reply:
Nevermind, I was looking at the wrong place. 🤦‍♂️